core-web/libs/edit-content/src/lib/fields/dot-edit-content-relationship-field/components/dot-select-existing-content/store/existing-content.store.ts

```diff
@@ -56,15 +54,34 @@ export const ExistingContentStore = signalStore(
              * Initiates the loading of content by setting the status to LOADING and fetching content from the service.
              * @returns {Observable<void>} An observable that completes when the content has been loaded.
              */
-            loadContent: rxMethod<void>(
+            loadContent: rxMethod<string>(
                 pipe(
                     tap(() => patchState(store, { status: ComponentStatus.LOADING })),
-                    switchMap(() =>
-                        relationshipFieldService.getContent().pipe(
+                    tap((contentId) => {
+                        if (!contentId) {
+                            patchState(store, {
+                                status: ComponentStatus.ERROR,
+                                errorMessage: 'dot.file.relationship.dialog.content.id.required'
+                            });
+                        }
+                    }),
+                    filter((contentId) => !!contentId),
+                    switchMap((contentId) =>
+                        relationshipFieldService.getColumnsAndContent(contentId).pipe(
                             tapResponse({
-                                next: (data) =>
-                                    patchState(store, { data, status: ComponentStatus.LOADED }),
-                                error: () => patchState(store, { status: ComponentStatus.ERROR })
+                                next: ([columns, data]) => {
+                                    patchState(store, {
+                                        columns,
+                                        data,
+                                        status: ComponentStatus.LOADED
+                                    });
+                                },
+                                error: () =>
+                                    patchState(store, {
```

Comment:
should we use here the `dotHttpErrorManagerService` to show a dialog to the user with the actual error ?

Reply:
I applied this `dotHttpErrorManagerService` in the `RelationshipFieldService`